.github/workflows/presubmit.yaml

```
@@ -25,24 +25,38 @@ jobs:
         run: |
           make _hadolint
 
-  build-and-test:
+  centos9-bootc-test:
     runs-on: ubuntu-latest
     steps:
       - name: Check out MicroShift upstream repository
         uses: actions/checkout@v4
 
-      - name: Compute the latest OKD version tag
-        id: okd-version-tag
-        shell: bash
-        run: |
-          sudo apt-get install -y jq
-          # Get the latest OKD version tag
-          okd_version_tag="$(curl -s https://quay.io/api/v1/repository/okd/scos-release/tag/ | jq -r ".tags.[].name" | sort | tail -1)"
-          echo "okd_version_tag=${okd_version_tag}" >> $GITHUB_OUTPUT
+      - name: Detect OKD version tag
+        id: detect-okd-version
+        uses: ./.github/actions/okd-version
+
+      - name: Run the build action
+        uses: ./.github/actions/build
+        with:
+          ushift-branch: main
+          okd-version-tag: ${{ steps.detect-okd-version.outputs.okd-version-tag }}
+          action: build-bootc-image
+
+  centos10-bootc-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out MicroShift upstream repository
+        uses: actions/checkout@v4
+
+      - name: Detect OKD version tag
+        id: detect-okd-version
+        uses: ./.github/actions/okd-version
 
       - name: Run the build action
         uses: ./.github/actions/build
         with:
           ushift-branch: main
-          okd-version-tag: ${{ steps.okd-version-tag.outputs.okd_version_tag }}
+          okd-version-tag: ${{ steps.detect-okd-version.outputs.okd-version-tag }}
+          bootc-image-url: quay.io/centos/centos
```

Otherwise the job name should be changed (remove `bootc`)
```suggestion
          bootc-image-url: quay.io/centos-bootc/centos-bootc
```